org.eclipse.lsp4e.test/src/org/eclipse/lsp4e/test/LanguageServiceAccessorTest.java

```diff
@@ -434,15 +434,15 @@ public void testSingletonLS() throws Exception {
 		var testFile1 = createFile(project, "shouldUseSingletonLS.lsp-singletonLS", "");
 		IDocument document1 = getDocument(testFile1);
 		assertNotNull(document1);
-		var languageServers = getLanguageServers(document1, MATCH_ALL);
+		var languageServers = getLanguageServers(document1, MATCH_ALL).get();
```

Comment:
I don't think this should be necessary here. The choice of using a singleton LS doesn't require that the LanguageServer is ready; this shouldn't be something we need for the test to pass. If this is required for the test to pass, then it probably shows underlying code has some bug.

Reply:
No, it is not necessary. I am currently just trying out things as I am getting a lot of very random test failures on GHA.